io/logs/logutil_test.go

```diff
@@ -24,3 +27,58 @@ func TestMaskCredentialsLogging(t *testing.T) {
 		require.Equal(t, MaskCredentialsLogging(test.url), test.maskedUrl)
 	}
 }
+
+func TestConfigurePersistantLogging(t *testing.T) {
```

Comment:
For temporary directories for tests use `t.TempDir()`  , this gets cleaned up by the test runner after the test is executed.